Documentation/WritingDocForExtension/CreateFromScratch.rst

```diff
@@ -108,13 +117,18 @@ Method 2: Start Documentation from Example Manual
    Before you publish your changes, make sure the documentation is rendered
    correctly.
 
-   Look at `Rendering Documentation With Docker <https://github.com/t3docs/docker-render-documentation/blob/master/README.rst>`__ for a quick start.
+   Look at :ref:`render-documenation-with-docker` for a quick start.
 
 6. When You Are Done, Publish Your Changes
 
    If you are working on your own extension, `make it publicly available
    <https://extensions.typo3.org/faq/publish-an-extension/>`__.
-   The documentation will then automatically be rendered on docs.typo3.org.
+
+.. important::
+
+   In order to trigger documentation rendering on the documentation server you should
```

Comment:
- should
+ have to

Reply:
I pushed another change which should fix this.